### DISCUSSION TITLE
is .net framework supported

### DISCUSSION THREAD

**Discussion by ahmedsza** [Azure/azure-dev#1827] [General]
Hello

Is .net framework (specifically asp.net) supported. I tried to have a sample app and it did not work.

**Comment by lechnerc77** [comment]
Can you elaborate a bit more on the issue? What is the error message (i guess when doing the `azd deploy`. Do you have a repository where one could look at the setup?

**Comment by ahmedsza** [comment]
Hi @lechnerc77 . Yes the DevOps starter is being deprecated and the recommendation is to use azd. So I tried to create a template that would mimic the devops starter for Windows (it was part of a lab). This used windows appservice and .net framework asp.net mvc app (just the base template out of the box). but a .net framework app with not work with dotnet build.. I did work around it right now by creating  a .net core based mvc solution - and that seems to work
I will look to create a sample repo and share it..
